.eslintrc.js

```diff
@@ -20,6 +20,7 @@ module.exports = {
     {
       files: ['*.md'],
       rules: {
+        'arrow-body-style': 0,
```

Comment:
markdown is not autofixable (although the markdown plugin got a PR today, so maybe soon! (https://github.com/eslint/eslint-plugin-markdown/pull/97))